### DISCUSSION TITLE
Welcome to quantum-tutor Discussions!

### DISCUSSION THREAD

**Discussion by Namasivaayam-L** [Quantum-Tutors/quantum-tutor#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by harisankar01** [comment]
Try to add `labels` for your PR and also assign it to the correct `milestone` for tracking. Thanks!!

**Reply by Namasivaayam-L** [reply]
I saw the option of Milestone what's it used for?

**Reply by harisankar01** [reply]
Tracking project deadline

**Reply by Namasivaayam-L** [reply]
oh okaY!
